mesh/mesh.sh

```diff
@@ -25,7 +25,7 @@ create_token() {
   nonce_count='001'
   timestamp=$(date +"%Y%m%d%H%M")
   hash_content="${MAILBOX_ID}:${nonce}:${nonce_count}:${MAILBOX_PASSWORD}:${timestamp}"
-  hash_value=$(echo -n "${hash_content}" | openssl dgst -sha256 -hmac "${SHARED_KEY}")
+  hash_value=$(echo -n "${hash_content}" | openssl dgst -sha256 -hmac "${SHARED_KEY}" | sed 's/^.* //')
```

Comment:
this will fix the problem when '(stdin)' is present in output.
example: (stdin)= 0beec7b5ea3f0fdbc95d0dd47f3c5bc275da8a33